test/k8sT/PoliciesNightly.go

```diff
@@ -48,8 +51,17 @@ var _ = Describe("NightlyPolicies", func() {
 
 	BeforeEach(func() {
 		once.Do(initialize)
+		err := kubectl.HeapsterDeploy()
+		Expect(err).To(BeNil(), "cannot deploy heapster")
+
 	})
 
+	MemoryProfiler := func(ctx context.Context) {
+		kubectl.CiliumExportInfo(ctx, "k8s_nightly_policies", map[string]string{
+			"endpoint_list": "kubectl get pods --all-namespaces | wc -l",
```

Comment:
This is incorrect. The output of `kubectl get pods --all-namespaces`:
```
NAMESPACE     NAME                          READY     STATUS              RESTARTS   AGE
kube-system   kube-addon-manager-minikube   1/1       Running             0          6s
kube-system   kube-dns-86f6f55dd5-dm4vc     0/3       ContainerCreating   0          1s
kube-system   kubernetes-dashboard-m2k2c    0/1       ContainerCreating   0          1s
kube-system   storage-provisioner           0/1       ContainerCreating   0          2s
```

Ideally you should be able to get this via JSON output? I want to avoid getting information from CLI that isn't formatted in some structured way that is subject to change without notice.

Reply:
Fixed.